x-pack/plugins/graph/index.js

```diff
@@ -30,6 +30,18 @@ export function graph(kibana) {
       home: ['plugins/graph/register_feature'],
       mappings,
       migrations,
+      savedObjectSchemas: {
+        'graph-workspace': {
```

Comment:
There's a bug where the index pattern name is saved as a reference and not the id, this breaks the ability to include dependencies for graph.